spec/support/features/test_features_helpers.rb

```diff
@@ -37,12 +35,17 @@ def click_add_widget
       first('.st-block-replacer').click
     end
 
+    def wait_for_sir_trevor
+      expect(page).to have_selector('.st-blocks.st-ready')
+      sleep 1
```

Comment:
Why do we need a sleep here?

Reply:
In theory we shouldn't. Sir Trevor adds the `st-ready` class right at the end of their initialization. Nevertheless, the rate of false alarms seems to increase without it. I don't have a good sense of if this is a Sir Trevor issue or if the extra second is giving some other source of CI unreliability the time it needs.

Reply:
This doesn't make sense to me.  How have you proved your assertion that the rate increases without this `sleep`?

Reply:
What is the current capybara timeout setting for this test?

Reply:
This is most of them:

https://github.com/projectblacklight/spotlight/blob/20b51cb9b21e3944f1d283bbe3e2298a032f32aa/spec/features/javascript/blocks/solr_documents_block_spec.rb#L3